### PR TITLE
Add an `INTERNAL` story for `_ComplianceBanner`

### PR DESCRIPTION
**Pull request by prprabhu-ms**
# What

https://user-images.githubusercontent.com/82062616/167750310-69269023-bca6-4c9d-96cd-4e23ad8969f4.mp4

# Why
This will allow us to experiment with the `_ComplianceBanner` implementation.

### CODE REVIEW

File: packages/storybook/stories/INTERNAL/ComplianceBanner.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+const ComplianceBannerStory = (): JSX.Element => {
+  const strings = _useCompositeLocale().strings.call;
+  const [callRecordState, setCallRecordState] = useState(true);
+  const [callTranscribeState, setCallTranscribeState] = useState(true);
+  return (
+    <Stack styles={containerStyles} tokens={stackTokens}>
+      <Stack horizontal tokens={stackTokens}>
+        <Checkbox
+          label="Recoding is in progress"
```
Comment (edwardlee-msft) [inline packages/storybook/stories/INTERNAL/ComplianceBanner.stories.tsx:21]: A small spelling error:
Recoding -> Recording

File: packages/storybook/stories/INTERNAL/ComplianceBanner.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+          checked={callRecordState}
+          onChange={(ev?: unknown, isChecked?: boolean) => setCallRecordState(!!isChecked)}
```
Comment (edwardlee-msft) [inline packages/storybook/stories/INTERNAL/ComplianceBanner.stories.tsx:23]: For those that may stumble upon this PR and have never seen the !! before,
it converts a truthy or falsey object to boolean. 
(e.g. 0, null, undefined, etc would be converted to false)

**Review by edwardlee-msft — APPROVED** [review]

File: packages/storybook/stories/INTERNAL/ComplianceBanner.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+// This must be the only named export from this module, and must be named to match the storybook path suffix.
+// This ensures that storybook hoists the story instead of creating a folder with a single entry.
+export const ComplianceBanner = ComplianceBannerStory.bind({});
+
+export default {
+  id: `${COMPONENT_FOLDER_PREFIX}-internal-compliancebanner`,
+  title: `${COMPONENT_FOLDER_PREFIX}/Internal/Compliance Banner`,
+  component: ComplianceBannerComponent,
+  argTypes: {
+    callRecordState: hiddenControl,
```
Comment (prabhjot-msft) [inline packages/storybook/stories/INTERNAL/ComplianceBanner.stories.tsx:61]: Rest of the code looks good. The `hidden control` is bit confusing, what is it and why are we using it. Aren't we controlling banner by provided checkboxes.

File: packages/storybook/stories/INTERNAL/ComplianceBanner.stories.tsx (hunk 1 above)
Comment (prprabhu-ms) [reply packages/storybook/stories/INTERNAL/ComplianceBanner.stories.tsx:21]: Heh, can't spell.

File: packages/storybook/stories/INTERNAL/ComplianceBanner.stories.tsx (hunk 3 above)
Comment (prprabhu-ms) [reply packages/storybook/stories/INTERNAL/ComplianceBanner.stories.tsx:63]: We need to hide the controls that storybook automatically generates for all the props.

Added a comment to that effect. Hopefully it helps clarify the code.

**Review by prabhjot-msft — APPROVED** [review]

**Pull request merged**